pages/linux/libreoffice.md

```diff
@@ -0,0 +1,20 @@
+# libreoffice
+
+> Use the powerful and free office suite, LibreOffice, from the command line.
+> More information: <https://www.libreoffice.org/>.
+
+- Open the given files in viewer mode (read-only):
+
+`libreoffice --view {{file1}} {{file2}} {{file3}}`
+
+- Dump text contents of the following files to console:
+
+`libreoffice --cat {{file1}} {{file2}} {{file3}}`
```

Comment:
```suggestion
`libreoffice --cat {{path/to/file1}} {{path/to/file2}}`
```

Reply:
The reason I did this format was because I saw it in the page for `tar`. Maybe mention this in the style guide?

Reply:
Yeah, we applied that format in `tar` to avoid long examples. But I think is ok here. wdyt?